src/test/java/net/snowflake/client/jdbc/SnowflakeDriverTest.java

```diff
@@ -115,7 +115,7 @@ public void testAcceptUrls() throws Exception {
             "localhost",
             8081,
             null,
-            expectedParameters));
+            expectedParameters));  */
```

Comment:
Why is this commented out? Shouldn't the test cases be added?

Reply:
I should have removed them but wanted to get your attention to it as well. These tests are in my opinion incorrect. The driver should never consider `https://localhost` as a valid URL. These tests ensure that this URL work. The changes I have introduced would consider this URL invalid.